docker_client.go

```diff
@@ -97,7 +97,7 @@ func splitDockerImage(img string) (string, string, string) {
 func getContainers(client *docker.Client) ([]*RuntimeContainer, error) {
 
 	apiContainers, err := client.ListContainers(docker.ListContainersOptions{
-		All:  false,
+		All:  true,
```

Comment:
I think this approach is fine as is for now.  I'm not sure I under how the `Generator` concept would work, but not opposed to that either.